crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -265,60 +266,26 @@ pub(crate) trait StateReadExt: StateRead + address::StateReadExt {
         Ok(Some(addr))
     }
 
-    #[instrument(skip_all)]
-    async fn get_deposit_nonce(&self, rollup_id: &RollupId) -> Result<u32> {
-        let bytes = self
-            .nonverifiable_get_raw(&deposit_nonce_storage_key(rollup_id))
-            .await
-            .map_err(anyhow_to_eyre)
-            .wrap_err("failed reading raw deposit nonce from state")?;
-        let Some(bytes) = bytes else {
-            // no deposits for this rollup id yet; return 0
-            return Ok(0);
-        };
-
-        let Nonce(nonce) =
-            Nonce(u32::from_be_bytes(bytes.try_into().expect(
-                "all deposit nonces stored should be 4 bytes; this is a bug",
-            )));
-        Ok(nonce)
-    }
-
     #[instrument(skip_all)]
     async fn get_deposit_rollup_ids(&self) -> Result<HashSet<RollupId>> {
```

Comment:
I'd like to replace this by a stream.

Reply:
i will work on this

Reply:
Happy to do this in a followup.  Ticket #1549.

Reply:
Yeah, dropped the ball on this one, already created a ticket for this here: https://github.com/astriaorg/astria/issues/1544

Reply:
I missed @noot's comment when I added mine.  Even happier to leave this to her :)